internal/netconf/interfaces.machine.tpl

```diff
@@ -4,8 +4,12 @@
 # See /etc/systemd/network for additional network configuration.
```

Comment:
The name of that file `interfaces.machine.tpl` reflects the type of os file it changes. Intenionally this was `/etc/network/interfaces`. If now a `.link` or a `.network` file is targeted the filename should follow that convention. E.g. `link.machine.tpl`.

Reply:
The same applies for `internal/netconf/testdata/interfaces.machine`

Reply:
OK, will do

Reply:
done